content/docs/guides/components/istio.md

```diff
@@ -45,14 +49,27 @@ gcloud container clusters describe XXXXXXX --zone=XXXXXX | grep -e clusterIpv4Ci
 clusterIpv4Cidr: 10.4.0.0/14
 servicesIpv4Cidr: 10.7.240.0/20
 ```
-5. Set the outbound IP ranges (direcly in the configmap, or use helm). The value should be like
+
+- Set the outbound IP ranges (direcly in the configmap). The value should be like
    `"10.32.0.0/14\,10.35.240.0/20"`.
 
-6. Install istio (without mTLS)
+### 4. Configure istio-ingressgateway
+We are using a regular ingress instead of Istio's ingressgateway as the entry to cluster, so this service should not
```

Comment:
typo "ingressgateway" -> "ingress gateway"

Should we clarify that on GCP we recommend using NodePort and GCLB but on other platforms you might want to use ISTIO ingress?

Reply:
done